src/es/isaac-penington.yml

```diff
@@ -0,0 +1,253 @@
+name: Isaac Penington
+slug: isaac-penington
+description: Isaac Penington (1616–1679) fue criado en una familia rica e influyente, y cultivado y pulido con una buena y amplia educación. Sin embargo, las aspiraciones de Penington, desde sus primeros días, estaban enteramente fijadas en las cosas de arriba y no en las muchas cosas del mundo que estaban a su alcance. Él dice de sí mismo — “Desde mi niñez mi corazón fue apuntado hacia el Señor, de quien me ocupé y a quien busqué desde mis tiernos años. Yo sentía que no podía estar satisfecho con (ni buscar) las cosas de este mundo perecedero, las cuales naturalmente expiran, sino que deseaba un verdadero sentido y unidad con lo que permanece para siempre.” De hecho, el hambre y la desesperación de Penington por el verdadero conocimiento y experiencia de Dios fue un sello distintivo en su vida. Buscando, encontró; llamando, la puerta se abrió delante de él. Buscó al Señor con todo su corazón, mente y fuerza, y no cabe duda de que el Señor fue encontrado por él.
+documents:
+  -
+    title: Los Escritos de Isaac Penington — Volumen 1
+    slug: escritos-volume-1
+    description: Isaac Penington (1616–1679) fue criado en una familia rica e influyente, y cultivado y pulido con una buena y amplia educación. Sin embargo, las aspiraciones de Penington, desde sus primeros días, estaban enteramente fijadas en las cosas de arriba y no en las muchas cosas del mundo que estaban a su alcance. Él dice de sí mismo — “Desde mi niñez mi corazón fue apuntado hacia el Señor, de quien me ocupé y a quien busqué desde mis tiernos años. Yo sentía que no podía estar satisfecho con (ni buscar) las cosas de este mundo perecedero, las cuales naturalmente expiran, sino que deseaba un verdadero sentido y unidad con lo que permanece para siempre.” De hecho, el hambre y la desesperación de Penington por el verdadero conocimiento y experiencia de Dios fue un sello distintivo en su vida. Buscando, encontró; llamando, la puerta se abrió delante de él. Buscó al Señor con todo su corazón, mente y fuerza, y no cabe duda de que el Señor fue encontrado por él.
+    tags:
+      - devotional
+      - doctrinal
+      - letters
+    editions:
+      -
+        type: modernized
+        pages: 391
+        formats:
+          -
+            type: softcover
+          -
+            type: audio
+          -
+            type: pdf
+          -
+            type: epub
+          -
+            type: mobi
+        audio:
+          reader: Jason Henderson
+          parts:
+            -
+              title: Part 1 # All of these audio parts are for the ENGLISH audiobook, and need to be redone for Spanish.
```

Comment:
Do the spanish recordings exist?  Are they on Soundcloud?  I'm wondering if we should either a) get the real recordings dialed in here (I can explain how I think), or b) remove this until we're able to put them here....